pennylane/ops/functions/iterative_qpe.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This file contains the function which generates the iterative QPE subroutine.
+"""
+
+import pennylane as qml
+import numpy as np
+
+
+def iterative_qpe(base, estimation_wire, iters):
+    r"""Function that is in charge of adding to the queue the necessary gates to implement iterative QPE.
+    It returns a list of the mid-circuit measurements performed.
+
+    Args:
+      base (Operator): the phase estimation unitary, specified as an :class:`~.Operator`
+      estimation_wire (Union[Wire, or int]): the wire to be used for the estimation.
```

Comment:
```suggestion
      estimation_wire (Union[Wire, int, str]): the wire to be used for the estimation.
```

Reply:
The wire label can be a string as well, also I think in your code examples you pass a list here